common/models.py

```diff
@@ -82,6 +83,14 @@ class Task(models.Model):
     plagiarism_key = models.CharField(max_length=255, null=True)
     created_at = models.DateTimeField(auto_now_add=True)
 
+    class TaskType(models.TextChoices):
+        # Add wanted task types here
+        HOMEWORK = ("HW", "Homework")
```

Comment:
Could you please make the field nullable and make the default `NULL`? We have 3k+ existing tasks and I want to distinguish tasks that don't have the type set yet, rather than setting a default for all tasks, which will be incorrect for some of them.

Reply:
Also, we don't need the (slightly confusing) shortcuts in the DB, let's just use "homework", "test", "project" :)